src/Raven.Client/Documents/Session/Tokens/IntersectToken.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Text;
+
+namespace Raven.Client.Documents.Session.Tokens
+{
+    public class IntersectToken : QueryToken
```

Comment:
Whatever makes this easier to resolve. Note that we can defer intersect support for now.